### DISCUSSION TITLE
Is time lapse the same as time delay?

### DISCUSSION THREAD

**Discussion by Parry29** [gopro/labs#907] [Q&A]
Hello everyone,

I am new to gopros and am trying to take an extended time lapse (24 hours) using this QR: https://[gopro.github.io/labs/control/longtimelapse/](https://gopro.github.io/labs/control/longtimelapse/)

The gopro scans the QR fine, but when it powers on to capture a picture, the screen changes to 360 photo (instead of time lapse). Is that by design? 

As far as I can tell, the images are being taken at the correct increments.

Thank you.

**Comment by dnewman-gpsw** [comment]
It is by design, as timelapse wouldn't last 24 hours.  With Timelapse modes the camera remains on between captures, but this script turns the camera off, allowing very long capture off a single battery charge.
